cmake/external/icu.cmake

```diff
@@ -74,11 +74,12 @@ endif()
 
 set(ICU_INCLUDE_DIRS "${ICU_INSTALL_DIR}/include")
 
-# Compile flags
+# Compile & link flags
 
 if(NOT WIN32)
   set(ICU_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wno-deprecated-declarations")
   set(ICU_C_FLAGS "${CMAKE_C_FLAGS} -fPIC -Wno-deprecated-declarations")
+  set(ICU_LINKER_FLAGS "${CMAKE_STATIC_LINKER_FLAGS} -ldl")
```

Comment:
please, cmake var https://cmake.org/cmake/help/latest/variable/CMAKE_DL_LIBS.html instead of hardcoded library

Reply:
Will
```
set(ICU_LINKER_FLAGS "${CMAKE_STATIC_LINKER_FLAGS} -l${CMAKE_DL_LIBS}")
```
be ok? CMAKE_DL_LIBS contains "dl" only

Reply:
yes